google/cloud/aiplatform/training_jobs.py

```diff
@@ -4453,110 +5265,53 @@ def _run(
-            }
-
-        if self._additional_experiments:
-            training_task_inputs_dict[
-                "additionalExperiments"
-            ] = self._additional_experiments
 
-        model = gca_model.Model(
-            display_name=model_display_name or self._display_name,
-            labels=model_labels or self._labels,
-            encryption_spec=self._model_encryption_spec,
-        )
+        Raises:
+            RuntimeError: If Training job has already been run or is waiting to run.
+        """
 
-        new_model = self._run_job(
-            training_task_definition=training_task_definition,
-            training_task_inputs=training_task_inputs_dict,
+        return super().run(
             dataset=dataset,
+            target_column=target_column,
+            time_column=time_column,
+            time_series_identifier_column=time_series_identifier_column,
+            unavailable_at_forecast_columns=unavailable_at_forecast_columns,
+            available_at_forecast_columns=available_at_forecast_columns,
+            forecast_horizon=forecast_horizon,
+            data_granularity_unit=data_granularity_unit,
+            data_granularity_count=data_granularity_count,
             training_fraction_split=training_fraction_split,
             validation_fraction_split=validation_fraction_split,
             test_fraction_split=test_fraction_split,
             predefined_split_column_name=predefined_split_column_name,
             timestamp_split_column_name=timestamp_split_column_name,
-            model=model,
+            weight_column=weight_column,
+            time_series_attribute_columns=time_series_attribute_columns,
+            context_window=context_window,
+            budget_milli_node_hours=budget_milli_node_hours,
+            export_evaluated_data_items=export_evaluated_data_items,
+            export_evaluated_data_items_bigquery_destination_uri=export_evaluated_data_items_bigquery_destination_uri,
+            export_evaluated_data_items_override_destination=export_evaluated_data_items_override_destination,
+            quantiles=quantiles,
+            validation_options=validation_options,
+            model_display_name=model_display_name,
+            model_labels=model_labels,
+            additional_experiments=additional_experiments,
+            sync=sync,
             create_request_timeout=create_request_timeout,
         )
 
-        if export_evaluated_data_items:
-            _LOGGER.info(
-                "Exported examples available at:\n%s"
-                % self.evaluated_data_items_bigquery_uri
-            )
-
-        return new_model
-
-    @property
-    def _model_upload_fail_string(self) -> str:
-        """Helper property for model upload failure."""
-        return (
-            f"Training Pipeline {self.resource_name} is not configured to upload a "
-            "Model."
-        )
-
     @property
     def evaluated_data_items_bigquery_uri(self) -> Optional[str]:
```

Comment:
You can just remove this if you call the super implementation.

Reply:
Do you know if removing this would impact the documentation?

Reply:
I think they can follow the superclass for docstrings.
Not sure if we use any automated tools that would be affected by this.

@sasha-gitg any thoughts? I would prefer less redundant code if possible.

Reply:
`nox -s docs` isn't creating documentation for `run()` as expected, since it's now part of the parent class. Looking into ways to force a child class to document a method from the parent class.

Reply:
Removed `evaluated_data_items_bigquery_uri`, moving discussion to the `run` method which I kept.